src/main/java/doore/study/domain/Study.java

```diff
@@ -51,4 +58,45 @@ public class Study extends BaseEntity {
     @OneToMany(mappedBy = "study")
     private List<CurriculumItem> curriculumItems;
 
+
+    public void createCurriculumItems(List<CurriculumItem> newCurriculumItems) {
+        if (curriculumItems == null) {
+            return;
+        }
```

Comment:
```suggestion
    @OneToMany(mappedBy = "study")
    private final List<CurriculumItem> curriculumItems = new ArrayList<>();


    public void createCurriculumItems(List<CurriculumItem> newCurriculumItems) {
```
컬렉션 필드는 불변을 위해 final로 지정하는 건 어떨까요?
+) 컬렉션 null 체크도 생략하는 건 어떨까요